package.json

```diff
@@ -29,7 +30,7 @@
     "@babel/plugin-transform-runtime": "^7.16.4",
     "@babel/preset-env": "^7.16.4",
     "@babel/runtime": "^7.16.3",
-    "@mapbox/mapbox-gl-draw": "1.3.0",
+    "@mapbox/mapbox-gl-draw": "1.4.1",
```

Comment:
mapbox-gl-draw is a dev dependency so that it's source files are accessible during the build. Source files are not part of the published package after the upgrade though.